Tools/TestWebKitAPI/Tests/WGSL/ParserTests.cpp

```diff
@@ -39,6 +39,32 @@
 #import "WGSL.h"
 #import <wtf/DataLog.h>
 
+#define ASSERT_BUILTIN(attr, attr_name) \
```

Comment:
why macro? why not function?

you can use `__file__` and `__line__` if necessary